{{cookiecutter.repo_name}}/requirements/base.txt

```diff
@@ -1,6 +1,9 @@
 django>=1.8.7,<1.9
+django_compressor==1.5
 django-extensions==1.5.5
+django-libsass==0.4
```

Comment:
Is this so that the cookie cutter always get the latest package? if so the should i remove the versions from all apps (except django)?